tedge_config/src/tedge_config.rs

```diff
@@ -1,6 +1,9 @@
 use crate::*;
 
-const DEFAULT_ROOT_CERT_PATH: &str = "/etc/ssl/certs";
+const DEFAULT_DEVICE_CERT_PATH: &str = "/etc/ssl/certs/tedge-certificate.pem";
+const DEFAULT_DEVICE_KEY_PATH: &str = "/etc/ssl/certs/tedge-private-key.pem";
```

Comment:
The default fir the key path is `/etc/tedge/device-certs/tedge-private-key.pem`.